docs/output.md

```diff
@@ -2,31 +2,78 @@
+10. Present QC for raw read, trimming, alignment, filtering and quantification ([`MultiQC`](http://multiqc.info/), [`R`](https://www.r-project.org/))
+
+## TrimGalore
+
+The nfcore/slamseq pipeline uses [TrimGalore](http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) for removal of adapter contamination and trimming of low quality regions. TrimGalore uses [Cutadapt](https://github.com/marcelm/cutadapt) for adapter trimming and runs FastQC after it finishes.
+
+MultiQC reports the percentage of bases removed by TrimGalore in the _General Statistics_ table, along with a line plot showing where reads were trimmed.
+
+## Slamdunk
 
-## FastQC
-[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your reads. It provides information about the quality score distribution across your reads, the per base sequence content (%T/A/G/C). You get information about adapter contamination and other overrepresented sequences.
+[Slamdunk](https://github.com/t-neumann/slamdunk) is a software to map and quantify nucleotide-conversion containing read sets with ultra-high sensitivity. The nfcore/slamseq pipeline uses Slamdunk for mapping SLAMseq datasets, calculating QC metrics and extracting both total and converted read counts for differential transcriptional output analysis.
 
-For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+**Output directory: `results/slamdunk`**
 
-> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality. To see how your reads look after trimming, look at the FastQC reports in the `trim_galore` directory.
+* `bam/*.{bam,bai}`
+  * The aligned and filtered BAM and BAI files
+* `vcf/*.vcf`
+  * The called SNPs for filtering T>C SNPs from `slamdunk snp`
+* `count/utrs/*tsv`
+  * The total and converted read quantifications on a UTR level ([format details](https://t-neumann.github.io/slamdunk/docs.html#tcount-file-format)).
+* `count/genes/*csv`
+  * The total and converted read quantifications summarised per gene.
 
-**Output directory: `results/fastqc`**
+## DESeq2
 
-* `sample_fastqc.html`
-  * FastQC report, containing quality metrics for your untrimmed raw fastq files
-* `zips/sample_fastqc.zip`
-  * zip file containing the FastQC report, tab-delimited data file and plot images
+[DESeq2](https://doi.org/10.1186/s13059-014-0550-8) is used to call differential transcriptional output between conditions to infer direct transcriptional targets. The nfcore/slamseq pipeline uses the total read counts to calculate the sizeFactors and then proceeds with the converted read counts for the remaining steps of the DESeq2 workflow.
 
+**Output directory: `results/deseq2`**
+
+* Directly in the output directory you will find one subfolder per `group`.
+
+**Output directory: `results/deseq2/<group>`**
+
+* `PCA.pdf`
```

Comment:
If possible, it's nice to give instructions on roughly what to look for in the pipeline outputs. Even better if you can add images of what it looks like (mostly for users thinking about running the pipeline). Best of all if you can give good and bad examples.

See the [atacseq](https://nf-co.re/atacseq/docs/output) and [eager](https://nf-co.re/eager/docs/output) documentation for good examples..

Reply:
Added some images and rough descriptions on how they should look like.